src/utils/resources/vm/utils/disk/size.ts

```diff
@@ -30,7 +30,7 @@ export const formatBytes = (rawSize: string, unit?: string): string => {
   }
   const size = hasNumber(rawSize);
   const sizeUnit = hasSizeUnit(rawSize) || unit;
-  const sizeUnits = ['B', 'KiB', 'MiB', 'GiB', 'TiB'];
+  const sizeUnits = ['B', 'Ki', 'Mi', 'Gi', 'Ti'];
```

Comment:
```suggestion
  const sizeUnits = ['', 'Ki', 'Mi', 'Gi', 'Ti'];
```

Reply:
`formatBytes` is implemented to use raw data, which should be without "B" suffix for the units, so it must have not been working nowhere (incl. _Disks_ tab), where used, if used properly. I'll check what's going on in the _Disks_ tab anyway, but adding "B" suffix to the units where needed, should be done in a followup PR, as this PR has nothing to do with it, and as the changes can affect more places than mentioned in this PR.

@upalatucci can you please, explain, why to remove 'B' from the `sizeUnits` array? I think we need it there.

Reply:
I think we should remove 'B' as it represents the unit byte and is not a prefix like Ki, Mi, Gi.

Reply:
You are right that "B" is not a prefix, but can't for example "1000B" come to this function as a raw size?

Reply:
No. According to [k8s](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/), you'll receive only '1000' without the B.

Reply:
> I think we should remove 'B' as it represents the unit byte and is not a prefix like Ki, Mi, Gi.

Ok, I will remove  'B' from that array, but not because it is not a prefix, but because we don't need it if we receive a raw size in bytes - a pure value without any unit, in `sizeUnits`. I really hope we won't receive anything like "30B" or so, in `sizeUnits`, because in that case it would not work.

Reply:
> But this change will bring back Gi quantities in the VM disk and filesystem tables.
> Can you add the B unit there?

Yes, I verified, that this PR will affect the units displayed in the VM _Disks_ tab - it is because `formatBytes` is used there, but this function's role is not displaying GiB instead of Gi (for example), but converting the raw value in bytes to something more user friendly. It was not fixed correctly, in one of the recent PRs, when trying to fix missing "B" when displaying units.

Anyway, fixing displaying units with "B" needs to be fixed separately. We need to add it when needed, differently than putting it to the `sizeUnits` array, because, as we can see, it causes bugs in other places. I have a solution but let's do it in a followup PR because this is not related to the BZ I am fixing in this PR. I'd like to use a nice [function](https://github.com/kubevirt-ui/kubevirt-plugin/pull/70/files#diff-4ebbb7ad9ddc308e1be2d3506502b5bed3bc53ed698243e291fbd840ea76838aR13) for adding "B", a function that also nicely checks if there is already "B" as part of the units, and if not, then adds it